cocotb/share/makefiles/simulators/Makefile.cvc

```diff
@@ -49,46 +49,39 @@ ifeq (, $(CMD))
     $(error "Unable to locate command >$(CMD_BIN)<")
 else
     CVC_BIN_DIR := $(shell dirname $(CMD))
-    export CVC_BIN_DIR
 endif
 
-#only interpreted mode works for the moment
-CVC_ITERP ?= 1
+CVC_ITERP := $(filter $(EXTRA_ARGS),+interp)
 
-ifeq ($(CVC_ITERP),1)
-    CVC_ARGS    += +interp
-endif
+ifneq (, $(CVC_ITERP)) # Interpreted mode
 
-# Compilation phase
-$(SIM_BUILD)/sim.vvp: $(SIM_BUILD) $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS)
-	MODULE=$(MODULE) \
-	TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        $(CMD) $(CVC_ARGS) +acc+2 -o $(SIM_BUILD)/sim.vvp +define+COCOTB_SIM=1 +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
+       $(COCOTB_RESULTS_FILE): $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS) $(CUSTOM_SIM_DEPS)
+		-@rm -f $(COCOTB_RESULTS_FILE)
 
-# Execution phase
-ifeq ($(CVC_ITERP),1)
-    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp
-else
-    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS)
-	MODULE=$(MODULE) \
-        TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        $(SIM_BUILD)/sim.vvp $(SIM_ARGS) $(EXTRA_ARGS) $(PLUSARGS)
-endif
+		MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
+		$(CMD) +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(SIM_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES) $(PLUSARGS)
+
+		$(call check_for_results_file)
 
-# Execution phase
-ifeq ($(CVC_ITERP),1)
-    debug:  $(CUSTOM_SIM_DEPS)
-	MODULE=$(MODULE) \
-        TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        gdb --args $(CMD) $(CVC_ARGS) +acc+2 -o $(SIM_BUILD)/sim.vvp +define+COCOTB_SIM=1 +loadvpi=$(LIB_DIR)/modelsim/libcocotbvpi:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
 else
-    debug: $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS)
-	MODULE=$(MODULE) \
-        TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) TOPLEVEL_LANG=$(TOPLEVEL_LANG) COCOTB_SIM=1 \
-        gdb --args $(SIM_BUILD)/sim.vvp $(SIM_ARGS) $(EXTRA_ARGS) $(PLUSARGS)
-endif
 
+    # Compilation phase
+    $(SIM_BUILD)/cvcsim: $(SIM_BUILD) $(VERILOG_SOURCES) $(CUSTOM_COMPILE_DEPS)
+	    $(CMD) +verbose -o $(SIM_BUILD)/cvcsim $(CVC_ARGS) +loadvpi=$(LIB_DIR)/libcocotbvpi_modelsim:vlog_startup_routines_bootstrap $(COMPILE_ARGS) $(EXTRA_ARGS) $(VERILOG_SOURCES)
+
+    # Execution phase
+    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/cvcsim $(CUSTOM_SIM_DEPS)
+		-@rm -f $(COCOTB_RESULTS_FILE)
```

Comment:
Indenting recipes like this seems like a recipe for mistakes. For other reviewers, the indentation here is:
```
    $(COCOTB_RESULTS_FILE): $(SIM_BUILD)/cvcsim $(CUSTOM_SIM_DEPS)
<tab><tab>-@rm -f $(COCOTB_RESULTS_FILE)
```

Reply:
Anything is fine for me. You prefer no intention for ifdef in this case ?

Reply:
Yeah, I think so - just leave blank lines around the `if` and `else` to make them easy to spot.